app/src/main/java/com/sun/android/menus_picker/AlertActivity.kt

```diff
@@ -0,0 +1,35 @@
+package com.sun.android.menus_picker
+
+import android.os.Bundle
+import android.provider.AlarmClock
+import android.widget.Toast
+import androidx.appcompat.app.AlertDialog
+import androidx.appcompat.app.AppCompatActivity
+import com.sun.android.databinding.ActivityAlertBinding
+
+
+class AlertActivity : AppCompatActivity() {
+    val binding by lazy { ActivityAlertBinding.inflate(layoutInflater) }
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(binding.root)
+        binding.button.setOnClickListener{
+            var myAlertBuilder: AlertDialog.Builder = AlertDialog.Builder(this)
+            myAlertBuilder.setTitle("Alert")
+            myAlertBuilder.setMessage("Click OK to continue, or Cancel to stop: ")
```

Comment:
các biến k thay đổi thì dùng val , có thể dùng . để set các thuôc tính liên tiếp ( Builder partern)
```suggestion
            var myAlertBuilder: AlertDialog.Builder = AlertDialog.Builder(this)
           .setTitle("Alert")
           .setMessage("Click OK to continue, or Cancel to stop: ")
```